Makefile.am

```diff
@@ -307,10 +311,12 @@ MARKDOWN_COMMON_DEPS = \
 man/man8/%.8 : man/%.8.md $(MARKDOWN_COMMON_DEPS)
 	rm -f $@
 	mkdir -p man/man8
+if HAVE_MD2MAN_ROFF
```

Comment:
I don't think you need this block, just remove the man page from man8_MANS should be sufficient.

Reply:
Yes, I believe you are right.

Reply:
@williamcroberts updated the branch removing the block. Thanks a lot for catching this.